truffle.js

```diff
@@ -8,5 +8,10 @@ module.exports = {
   },
   mocha: {
     enableTimeouts: false
+  },
+  compilers: {
```

Comment:
Should we do the same for our uFragments config?

Reply:
Yes.